CONTRIBUTING.md

```diff
@@ -24,12 +24,12 @@ $ANDROID_HOME/tools/bin/sdkmanager "system-images;android-29;google_apis;x86"
 
 ## Setup
```

Comment:
I gues this is a verb too?

Reply:
I decided it wasn't. It's a noun heading describing the setup process.

Reply:
Fair